app/models/measure_condition.rb

```diff
@@ -122,22 +122,10 @@ def entry_price_system?
   end
 
   def expresses_unit?
-    condition_measurement_unit_code || measure_condition_components.any?(&:expresses_unit?)
+    measure_condition_components.any?(&:expresses_unit?)
   end
 
   def units
-    if condition_measurement_unit_code
```

Comment:
The condition_measurement_unit_code is for trigger price indications as part of the trigger price system and isn't used to calculate duties